corehq/apps/export/templates/export/partials/table.html

```diff
@@ -409,6 +411,7 @@ <h4 data-bind="text: case_type"></h4>
       </td>
     {% endif %}
 
+    {% if not is_odata %}
```

Comment:
Shouldn't this go with the `if has_edit_permissions` condition below? Or do you want to hide the export button itself?

Reply:
there is actually no export button for odata feeds. but I made the change for clarity. The entire column isn't relevant at all tho...